src/components/drag-and-drop-list/DragAndDropList.tsx

```diff
@@ -0,0 +1,137 @@
+import {useRef, useState} from "react";
+import "./drag-and-drop-list.css";
+import {DragAndDropTodoItem} from "../drag-and-drop-todo-item/DragAndDropTodoItem";
+import {GhostItem} from "../ghost-item/GhostItem";
+
+interface DragAndDropListProps {
+  title: string
+  listItems: Array<string>
+  listId: number
+  setListItems: (listItems: Array<string>) => void;
+  onItemDropped?: (item: string, sourceListId: number, targetListId: number) => void;
+}
+interface Coordinates {
+  x: number
+  y: number
+}
+
+export function DragAndDropList(props: DragAndDropListProps) {
+  const { title, listItems: items, setListItems: setItems, listId, onItemDropped } = props;
```

Comment:
Nit: Another way for this is
```suggestion
export function DragAndDropList({ title, listItems: items, setListItems: setItems, listId, onItemDropped } : DragAndDropListProps) {

```